new-backend/start.sh

```
@@ -0,0 +1,3 @@
+uv sync
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)